list.ftd

```diff
@@ -20,38 +20,48 @@ Here we have declared a new variable called `primes`, which is a `list` of
 
 -- ft.h1: Adding values to list
 
-The name of `list` can be used to insert more values in that list:
+After initializing the list, we can add elements to the list
+by mentioning the `list-type` like this: `-- <list-type>: <value>`.
+
+`Note`: Also we need to use the end syntax `-- end: <list-name>` to mark
+the end of the list.
```

Comment:
@Heulitig have you verified this? I think variables are immutable unless they start with $.

Reply:
> @Heulitig have you verified this? I think variables are immutable unless they start with $.

Here the list is immutable after its initialized. I have written this based on the elements which are added in the list during initialization. I need to rephrase the wording here to be more precise.